pwa/src/styling/design-tokens/component-overrides.css

```diff
@@ -317,3 +317,7 @@
   block-size: max-content;
   inline-size: max-content;
 }
+
+.utrecht-button-link--secondary-action:hover {
+  --utrecht-button-secondary-action-hover-background-color: #006e32;
+}
```

Comment:
Drie problemen:
- Je voegt hier hardcoded kleuren toe, in plaats van Rotterdam CSS variables te gebruiken voor het kleurenpalet
- je voegt hier hardcoded huisstijl toe aan de algemene component overrides
- je voegt CSS variables toe, terwijl het handiger is een PR te maken voor de Rotterdam theme in de themes repo